<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogStereoSettings</class>
 <widget class="QDialog" name="dialogStereoSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialogStereoSettings</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Coin3D library cant toggle between raw stereo and other stereo settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Defaults  menu to do this</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="offButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Stereo off</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rawButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Raw stereo (OpenGL Quadbuffered)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="anaglyphButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Anaglyph (red-blue)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="swapBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Swap Eyes -- if stereo effect 'inside out'</string>
   </property>
  </widget>
  <widget class="QSlider" name="offsetSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera offset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="offEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>offButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogStereoSettings</receiver>
   <slot>doOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rawButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogStereoSettings</receiver>
   <slot>doRaw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>anaglyphButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogStereoSettings</receiver>
   <slot>doAnaglyph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogStereoSettings</receiver>
   <slot>doOffset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>swapBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dialogStereoSettings</receiver>
   <slot>doSwap(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogStereoSettings</receiver>
   <slot>doCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>dialogStereoSettings</receiver>
   <slot>doOffsetEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doOff()</slot>
  <slot>doRaw()</slot>
  <slot>doAnaglyph()</slot>
  <slot>doSide()</slot>
  <slot>doOffset()</slot>
  <slot>doSwap(bool)</slot>
  <slot>doCancel()</slot>
  <slot>doOffsetEdit()</slot>
 </slots>
</ui>
